lib/ydwg02.js

```diff
@@ -43,23 +47,66 @@ function Ydwg02Stream (options) {
 
   this.fromPgn.on('error', (pgn, error) => {
     debug(`[error] ${pgn.pgn} ${error}`)
-  })  
+  })
+
 
-  /*
   if ( this.options.app ) {
     options.app.on('nmea2000out', (msg) => {
-      that.sendActisensePGN(msg)
+      that.sendYdwgPGN(msg)
     })
+
     options.app.on('nmea2000JsonOut', (msg) => {
       that.sendPGN(msg)
     })
 
-    this.sendString('$PDGY,N2NET_OFFLINE')
+    //this.sendString('$PDGY,N2NET_OFFLINE')
 
     debug('started')
     //this.options.app.emit('nmea2000OutAvailable')
   }
-  */
+
+}
+
+Ydwg02Stream.prototype.sendString = function (msg) {
+  debug('sending %s', msg)
+  this.options.app.emit('ydwg02-out', msg)
+}
+
+Ydwg02Stream.prototype.sendPGN = function (pgn) {
+  if ( this.cansend ) {
```

Comment:
`cansend` is not needed with this device, remove the check

Reply:
It has been updated.